src/app/services/cohort.service.ts

```diff
@@ -34,18 +38,25 @@ export class CohortService {
   private _currentCohort: Cohort;
   // The list of cohorts saved by the user
   private _cohorts: Cohort[] = [];
-  // The total numbers of subjects & observations
+  // The total numbers of subjects & observations for the current cohort
   private _totalCounts: CountItem;
```

Comment:
maybe it's better to rename it to sth like `_currentCohotCounts` then? `totalCounts` and `allCounts` are a bit confusing now

Reply:
true, will do that

Reply:
I removed the totalCounts all together, since I just realized it is not used.